azure-pipelines.yaml

```diff
@@ -7,4 +7,7 @@ pool:
 jobs:
   - template: build-and-push-template.yaml
     parameters:
-      subfolder: connect
\ No newline at end of file
+      subfolder: connect
+  - template: build-and-push-template.yaml
```

Comment:
Seems like the job name cannot have hyphens. Not sure how to solve that, but the pipeline is failing...